gsa/src/web/pages/tickets/createdialog.js

```diff
@@ -24,6 +24,7 @@ import SaveDialog from 'web/components/dialog/savedialog';
 
 import FormGroup from 'web/components/form/formgroup';
 import Select from 'web/components/form/select';
+import TextArea from 'web/components/form/textarea';
```

Comment:
IIRC we normally use TextField for adding/editing comments for entities. For consistency reasons, I'd do it here as well.

Reply:
doesn't make sense here. the normal comments are shorter. here we have longer comments that may contain newlines. texfields don't allow for displaying newlines.